streamer/proxy_node.py

```diff
@@ -228,15 +229,15 @@ def __init__(self, bucket: Any, base_path: str,
                rate_limiter: RateLimiter, *args, **kwargs) -> None:
     self._bucket: google.cloud.storage.Bucket = bucket
     self._base_path: str = base_path
-    self._chunked_output: Optional[BinaryIO] = None
+    self._chunked_output: Optional[Union[BinaryIO, BufferedIOBase]] = None
```

Comment:
i suggested the prev solution because i thought the issue stems from here. seems this is not that case though, why is this needed?

Reply:
I did a search-and-replace on BinaryIO.  I'll double-check if this is needed.

Reply:
Yeah, this one is irrelevant.  I'll revert it.

Reply:
Done